### PR TITLE
Bug 2087603: Align single vm details card with updated design

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

Align the single VM overview page- details card, follow a design update.

## 🎥 Demo

vnc-console-disabled

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/VirtualMachinesOverviewTabDetailsConsoleConnect.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+type VirtualMachinesOverviewTabDetailsConsoleConnectProps = {
+  connect?: () => void;
+};
+
+const VirtualMachinesOverviewTabDetailsConsoleConnect: React.FC<
+  VirtualMachinesOverviewTabDetailsConsoleConnectProps
+> = ({ connect }) => {
+  return (
+    <div className="vnc-grey-background">
+      <PlayIcon color="blue" size="md" onClick={connect} />
```
Comment (yaacov) [inline src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/VirtualMachinesOverviewTabDetailsConsoleConnect.tsx:14]: a. color should be from patternfly pallern
b. when vm is off, it should be disabled ( gray ? )

File: src/utils/components/Consoles/components/vnc-console/VncConsole.tsx [hunk 2]
```diff
@@ -175,16 +176,19 @@ export const VncConsole: React.FC<VncConsoleProps> = ({
...
-            <EmptyStateBody>
-              {textDisconnected || t('Click Connect to open the VNC console.')}
-            </EmptyStateBody>
-            <Button variant="primary" onClick={connect}>
-              {textConnect || t('Connect')}
-            </Button>
-          </EmptyState>
-        )}
+        {status === disconnected &&
+          (CustomConnectComponent ? (
```
Comment (yaacov) [inline src/utils/components/Consoles/components/vnc-console/VncConsole.tsx:180]: do we plan to use this component with a different way to re-connect ?

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/VirtualMachinesOverviewTabDetailsConsoleConnect.tsx (hunk 1 above)
Comment (metalice) [reply src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/VirtualMachinesOverviewTabDetailsConsoleConnect.tsx]: replaced with css var and added disabled apperance

File: src/utils/components/Consoles/components/vnc-console/VncConsole.tsx (hunk 2 above)
Comment (metalice) [reply src/utils/components/Consoles/components/vnc-console/VncConsole.tsx:180]: yes, console tab is different. and in the future, it's possible other places if needed will be custom as well.

**Pull request merged**